packages/eslint-plugin-template/src/rules/accessibility-valid-aria.ts

```diff
@@ -0,0 +1,60 @@
+const { aria } = require('aria-query');
```

Comment:
Please use an import (you'll need to install `@types/aria-query` as a devDep)

Reply:
Updated.